datasets/il/wmd_banned/il_wmd_sanction.yml

```diff
@@ -0,0 +1,47 @@
+title: Israeli Ministry of Finance Sanctioned Entities
+entry_point: crawler.py
+prefix: il-sanctioned
+coverage:
+  frequency: daily
+  start: 2024-09-12
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >-
+  List of entities and individuals subject to sanctions under the Prevention of Distribution and Financing of Weapons of Mass Destruction Law
+description: |
+  The list includes entities and individuals 
+   - Designated by the UN Security Council under Resolutions 2231 and 1718.
+   - Those restricted under the Trading with the Enemy Ordinance – 1939, and the Prevention of Distribution and Financing of Weapons of Mass Destruction Law – 2018. 
+   
+   Economic activity with these parties is restricted or forbidden under Israeli law.
+
+publisher:
+  name: Ministry of Finance
+  description: |
+    The Israeli Ministry of Finance is the primary economic ministry in Israel, overseeing the government's economic policies, fiscal targets, and budget management. 
+    It handles state revenues, tax collection, nonresident investments, and international economic relations.  [Read More](https://en.wikipedia.org/wiki/Ministry_of_Finance_(Israel))
+  country: il
+  url: https://www.gov.il/en/departments/ministry_of_finance/govil-landing-page
+  official: true
+url: https://www.gov.il/en/pages/declared_elements_list
+data:
+  url: https://www.gov.il/BlobFolder/generalpage/declared_elements_list/en/sanctions_headquarters_declared_elements_list-file2.xlsx
+  format: XLSX
+lookups:
+  type.country:
+    lowercase: true
+    normalize: true
+    options:
+      - contains:
+          - Democratic People's Republic of
+        value: kp
+      - contains:
+          - Iran
+        value: ir
+      - contains:
+          - 1969
+        value: null
+assertions:
+  min:
+    schema_entities:
+      Person: 100
+      Organization: 100
```

Comment:
let's add a max to this too - the max is helpful to discover with then min isn't sufficiently large any more.